### ISSUE TITLE
RCS Balance

### ISSUE THREAD

**Issue by pjf** [KSP-RO/RP-0#51]
Right now we have RCS parts, but they're not at all balanced. Let's look at some lower-tech examples. All thrust and Isp ratings are for HTP:

RCS Block (1/4):   $ 15 (0.06, 47-137, 2 axis)
RCS Block (1/2):   $ 20 (0.13, 47-137, 2 axis)
Mercury Roll Pack: $ 50 (0.11, 47-137, 1 axis)
1kN Thruster:      $100 (1.00, 47-137, ½ axis)
Vernor Engine:     $150 (0.26, 47-137, ½ axis)

Note that even though the roll-pack costs wice as much as the 1/2 RCS block, it provides less thrust, and in fewer directions. The Vernor engine costs an order of magnitude more, provides only twice as much thrust, and can only be used in a single direction. Indeed, there's not much reason to ever use anything other than the 1/2 RCS block.

There are ways we can balance this. The roll-pack can be dropped in price to ~ $15, where it becomes a better deal than the RCS blocks if you want single-axis control.

The Vernor engine claims it can use fuel+oxidiser mixes, and it can. With fuel+NTO it gives an impressive 0.45 thrust. However the RCS blocks can *also* use fuel+NTO, so it doesn't have a real advantage here.

If the RCS blocks were restricted to monopropellant only, and the Vernor was cut in price or boosted in power, it would then be more competitive. But right now, it's a terrible deal for everyone.

Alernatively, if we put the Vernor at the $20 price point it now has a real purpose. It provides double the thrust for the same cost, but only in one direction.

The 1kN thruster seems to be properly priced in the scheme of things, although the LR-101 feels like a better deal. The LR-101 is half the price, can gimbal, provides 5 times the thrust, and has comparble Isp. It's only downside is boil-off. At the very least I feel the 1kN thruster should allow throttling.

**TL;DR: There's no reason to use anything other than the 1/2 RCS block**

**Comment by NathanKell** [comment]
RCS prices across the board should come up; also, the individual CONFIGs should have separate costs (and be techRequired so you can't do biprop early).

**Comment by NathanKell** [comment]
I believe this has been handled RO-side, though we might consider making regular RCS cost more as well. Closing for now...
